### PR TITLE
feat(backend): 🍰 Email Templates

### LINKED ISSUE #1288: 🔧 [Refactor] Email Templates
## 🔧 Refactor ticket
 - find a library for HTML email templates
   - https://github.com/forwardemail/email-templates
   - See solution in [ocelot.social](https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/backend/src/middleware/helpers/email)
- refactor the localization of email subject, body and from
   - See solution in [ocelot.social](https://github.com/Ocelot-Social-Community/Ocelot-Social/tree/master/backend/src/middleware/helpers/email)

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Email templates.

### Issues

- fixes #1288

### Todo

- [x] questions:
  - [x] add a config to open send emails in the browser?
    - [x] the emails content is logged out anyways
    - [x] on macOS I get an error openning Firefox …
  - [x] do we want to have `HTML` and `text` in the mails?
    - is automatically at our configuration
  - [x] email content:
    - [x] `sendAccountMultiRegistrationEmail`:
      - [x] whos name shall appeare in the salutation:
        - [ ] ~~the name of the new registrant?~~
        - [x] or the name of the registered owner of the email?
- [x] set `i18n` logger functions?
  - [ ] but I haven't found if it works
- [ ] replace `i18n` `__` function by `t`?
- [x] set `i18n` language globaly for a user?
  - [x] is the language handled separately for each user really?
    - @Mogge says yes
  - [x] on create user
  - [x] on update user
  - [x] on login user
- [x] seeding is hanging on its end
  - this happens if we set `autoReload: true` on `i18n` in `createServer`
- [x] add `yarn locales --fix` for the backend? -> issue #2366 
- [ ] tests:
  - [ ] test `backend/src/graphql/resolver/UserResolver.test.ts`
    - [ ] call of `i18n.setLocale()`
  - [x] writing new ones
  - [x] have a look at `sends an account multi registration email` in file `src/graphql/resolver/UserResolver.test.ts`
    - because sometimes on my macOS it opens the browser and the iPhone simulator to preview the send mail
- [x] remove the old `sendAccountMultiRegistrationEmail` email
- [x] remove `backend/src/mailer/text/accountMultiRegistration.ts`
- [x] add good descriptions to the readme or delete it
- [x] cleanup: search for `Wolle`

### CODE REVIEW

File: backend/src/emails/sendEmailTranslated.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    // },
+    transport,
+    // uncomment below to open send emails in the browser
+    // preview: {
+    //   open: {
+    //     app: 'firefox',
+    //     wait: false,
+    //   },
+    // },
+    // i18n,
```
Comment (Tirokk) [inline backend/src/emails/sendEmailTranslated.ts:63]: `i18n` seems not needed …

File: admin/.env.dist [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CONFIG_VERSION=v1.2022-03-18
+CONFIG_VERSION=v10.2022-09-20
```
Comment (Tirokk) [inline admin/.env.dist:1]: Is this correct this way?
See other occations as well …

Why this is not in the master?
Do I understand something wrong?

File: backend/src/emails/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Using  `forwardemail`–`email-templates` With `pug` Package
```
Comment (ulfgebhardt) [inline backend/src/emails/README.md:1]: This documentation is not needed?! -> All steps are done, what is this for?

File: backend/src/emails/sendEmailTranslated.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import { backendLogger as logger } from '@/server/logger'
```
Comment (ulfgebhardt) [inline backend/src/emails/sendEmailTranslated.ts:1]: this functionality should be replacing the existing sendMail to have backward comptibility.
-> All untranslated/not html emails should be send the old way
-> All already properly defined emails should be sent the new way

**Review by ulfgebhardt — DISMISSED** [review]

File: backend/src/emails/sendEmailTranslated.ts (hunk 4 above)
Comment (Tirokk) [reply backend/src/emails/sendEmailTranslated.ts:1]: That's what I have in mind too! 🤗

File: backend/src/emails/sendEmailVariants.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+export const sendAccountMultiRegistrationEmail = (data: {
+  firstName: string
+  lastName: string
+  email: string
+  language: string
+}): Promise<boolean> => {
+  return sendEmailTranslated({
+    receiver: { to: `${data.firstName} ${data.lastName} <${data.email}>` },
+    template: 'accountMultiRegistration',
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply backend/src/emails/sendEmailVariants.ts:12]: Here are just the new ones.
The old ones are in `backend/src/mailer` and will be replace by the new ones and therefore getting removed if the replacement is in use.

File: backend/src/emails/accountMultiRegistration/html.pug [hunk 6]
```diff
@@ -0,0 +1,22 @@
+doctype html
+html(lang="en")
```
Comment (Mogge) [inline backend/src/emails/accountMultiRegistration/html.pug:2]: Follow issue. The language attribute must be set by the given locale. If not, we will have wrong hyphenation

Comment (Mogge) [reply backend/src/emails/accountMultiRegistration/html.pug:2]: As we send only German emails so far, please set the language to de

Comment (Tirokk) [reply backend/src/emails/accountMultiRegistration/html.pug:2]: I have tried to set this, but was not successful til now.
I will try one time again.

File: backend/src/emails/README.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+- [email-templates](https://github.com/forwardemail/email-templates)
+- [pug](https://www.npmjs.com/package/pug)
+
+## Installation
+
+To install both packages with yarn run:
+
+```bash
+yarn add email-templates pug
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply backend/src/emails/README.md:13]: This isntallation instructions can be removed I guess.

[the remark this answers is not in the document]
Comment (Tirokk) [reply backend/src/emails/README.md]: Removed that

**Review by Mogge — APPROVED** [review]

**Pull request merged**